tdrs-frontend/Dockerfile.local

```diff
@@ -5,33 +5,20 @@
 FROM node:12.18-alpine3.12 AS localdev
 
 RUN mkdir /home/node/app/ && chown -R node:node /home/node/app
-WORKDIR /home/node/app
-COPY --chown=node:node package.json yarn.lock ./
-USER node
-RUN yarn install && yarn cache clean
-COPY --chown=node:node . .
 
+WORKDIR /home/node/app
 
+USER node
 
-# ---
-# Stage 1: Create a production build
-FROM localdev AS build
-ENV SASS_PATH=node_modules
-RUN yarn build
+COPY --chown=node:node package.json yarn.lock ./
 
+RUN yarn install
 
+COPY --chown=node:node . ./
 
+ENV SASS_PATH=node_modules:src
+ENV PORT=80
```

Comment:
Good call on setting the port to 80 here, keeps thing a little simpler with the compose files.